pkgs/by-name/hn/hn-text/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "hn-text";
+  version = "0.1.0";
```

Comment:
just letting you know that latest version is 0.1.2 which has some important bugfixes :)